crates/core/src/engine.rs

```diff
@@ -88,8 +91,14 @@ impl FoyerEngine {
             })
             .with_shards(parallelism)
             .with_eviction_config(FifoConfig::default())
-            .storage(storage)
-            .with_device_options(dev)
+            .storage()
+            .with_engine_config(BlockEngineBuilder::new(dev))
```

Comment:
The previous code set file size with `.with_file_size(64 * 1024 * 1024)`. With v0.19.0, please set it here with `.with_block_size(64 * 1024 * 1024)` with `BlockEngineBuilder`. 

In the new version, the file size and block engine block size have been decoupled.

Reply:
Fixed.